lib/codegen/DecisionParser.cpp

```diff
@@ -16,11 +16,12 @@ class DTPreprocessor {
   KOREObjectSymbol *dv;
 
   enum Kind {
-    Switch, SwitchLit, Function, Leaf, Fail, Swap
+    Switch, SwitchLit, EqualsLiteral, Function, Leaf, Fail, Swap
```

Comment:
I'm not a fan of the inconsistent naming between SwitchLit and EqualsLiteral. Let's standardize on Lit or Literal (fine with either).